js/index.js

```diff
@@ -1,6 +1,60 @@
-// Iteration 1: Names and Input
+//console.log("I´m ready!")
 
-// Iteration 2: Conditionals
+let hacker1 = "Ana"
+  console.log(`The driver's name is ${hacker1}`)
 
+let hacker2 = "Tobias";
+    console.log(`The navigator's name is ${hacker2}`)
 
-// Iteration 3: Loops
+if (hacker1.length === hacker2.length){
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
+else if (hacker1.length < hacker2.length){
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+
+}
+else if (hacker1.length > hacker2.length){
+  console.log(`The driver has the longest name, it has ${hacker1.length}characters.`)
+}
+
+ 
+    
+let driverName = hacker1.split("").join(" ");
```

Comment:
Super work, so it is a super solution later. Actually it was meant that you use a for loop here (for now)

Reply:
let driverCapitals = ""

for (let i = 0; i < hacker1.length; i++) {
  
  let upperCaseLetter = hacker1[i].toUpperCase()
  
  // if it is the first letter to be added, then add it without the space
  if (i === 0) {
    driverCapitals = driverCapitals + upperCaseLetter
  } else {
    driverCapitals = driverCapitals + " " + upperCaseLetter
  }
  
}

console.log(driverCapitals)

like this for example